frontend/assets/javascripts/src/modules/form/validation/existEmail.es6

```diff
@@ -0,0 +1,31 @@
+import ajax from 'ajax';
+import debounce from 'lodash/debounce';
+
+export function init() {
+    const emailField = document.getElementById('email');
+    const isUseErrorMessage = document.getElementById('form-field__error-message-email-checker');
+    const checkerURL = '/user/check-existing-email';
+
+    const validateEmail = debounce(() => {
+        let prospectiveEmail = emailField.value;
+
+        if(!prospectiveEmail.includes('@') || !prospectiveEmail.includes('.')) {
```

Comment:
Pedantic maybe, but it is possible to have a valid email address that doesn't have any `.`s.  A number of country tlds have MX records (e.g. `.ai`, so `test@ai` may be a valid address)